AEPMessaging/Tests/TestHelpers/TestableMessagingMobileParameters.swift

```diff
@@ -53,7 +53,9 @@ class TestableMobileParameters {
         var eventData: [String: Any] = [
             MessagingConstants.Event.Data.Key.TRIGGERED_CONSEQUENCE: [
                 MessagingConstants.Event.Data.Key.DETAIL: [
-                    MessagingConstants.Event.Data.Key.IAM.MOBILE_PARAMETERS: mobileParameters
+                    MessagingConstants.Event.Data.Key.DATA: [
+                        "mobileParameters": mobileParameters
```

Comment:
nit: We are using constants for some variables and not for others e.g. "mobileParameters".